CONTRIBUTING.md

```diff
@@ -184,11 +184,11 @@ Modules tests are inside `test/modules/` directory. Currently, every module has
 
 ### 1. Release the newest version
 
-Go to the Actions tab and click the "Release Newest Version" tab. Then click the "Run Workflow" button to mark the newest version as released.
+Go to the Actions tab and click the ["Version: Release Newest" tab](https://github.com/iasql/iasql/actions/workflows/version.yml). Then click the "Run Workflow" button to mark the newest version as released.
 
 ### 2. Begin development of the next version
```

Comment:
Should we include the `Deploy to Production` step in here or no?

Reply:
yes